code/datums/ruins/lavaland.dm

```diff
@@ -44,12 +44,11 @@
 	cost = 10
 	allow_duplicates = FALSE
 
-/datum/map_template/ruin/lavaland/ash_walker
-	name = "Ash Walker Nest"
-	id = "ash-walker"
-	description = "A race of unbreathing lizards live here, that run faster than a human can, worship a broken dead city, and are capable of reproducing by something involving tentacles? \
-	Probably best to stay clear."
-	suffix = "lavaland_surface_ash_walker1.dmm"
+/datum/map_template/ruin/lavaland/dwarven_slumber
+	name = "Dwarven Awakening Spot"
+	id = "dwarf"
+	description = "Millenia ago dwarves ruled lavaland till a cataclysm came. Lucky ones sealed themselves inside a magical sarcophagus. First of the dwarves have awakened even tough the sarcophagus is damaged."
```

Comment:
spelling

Reply:
Improved description with changes and additions
```suggestion
	description = "Millennia ago, dwarves ruled the lavalands, till a cataclysm came. The lucky ones sealed themselves inside a magical sarcophagus, and slept for thousands upon thousands of years. The first dwarves have now awakened from their slumber, even if the sarcophagus had been damaged due to time."
```